<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_Input">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>661</width>
      <height>211</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Output">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>661</width>
      <height>211</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mes">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>560</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Port">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Adress">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Our_Port">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Port">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Порт </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Adress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Our_Port">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Наш порт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Create">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SendMes">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>560</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>224</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSpam">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>СПАМ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Address">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>439</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Address">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести IP адрес</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_Port</tabstop>
  <tabstop>lineEdit_Adress</tabstop>
  <tabstop>lineEdit_Our_Port</tabstop>
  <tabstop>pushButton_Create</tabstop>
  <tabstop>checkBoxSpam</tabstop>
  <tabstop>lineEdit_Mes</tabstop>
  <tabstop>pushButton_SendMes</tabstop>
  <tabstop>textEdit_Output</tabstop>
  <tabstop>textEdit_Input</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
